towhee/functional/mixins/milvus_mixin.py

```diff
@@ -19,11 +19,13 @@
 from ..entity import Entity
 
 
-def _milvus_insert(iterable: Iterable, index: Tuple[str], collection, batch_size: int): # pragma: no cover
+def _milvus_insert(iterable: Iterable, index: Tuple[str], host, port,
+                   collection_name, batch_size: int = 100):  # pragma: no cover
```

Comment:
@shiyu22 has implemented a url for milvus db, can we reuse the url here?

Reply:
Yes, you can refer to it to [parse uri](https://github.com/towhee-io/towhee/blob/main/towhee/connectors/ann_index/milvus.py#L58).